cmake/Platform/Mac/runtime_install_mac.cmake.in

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+cmake_path(SET file_path "${CMAKE_INSTALL_PREFIX}/@install_relative_binaries_path@/@runtime_output_filename@")
+
+if(@runtime_output_filename@ MATCHES ".app")
```

Comment:
To make sure that ".app" only matches at the end, the regular expression can use the end boundary anchor `.app$`